lms/templates/student_profile/learner_profile.underscore

```diff
@@ -0,0 +1,25 @@
+<div class="profile <%- ownProfile ? 'profile-self' : 'profile-other' %>">
+    <div class="wrapper-profile-field-account-privacy"></div>
+
+    <div class="wrapper-profile-sections account-settings-container">
+        <div class="wrapper-profile-section-one">
+            <div class="profile-photo">
+                <img src="<%- profilePhoto %>">
+            </div>
+
+            <div class="profile-section-one-fields">
+            </div>
+        </div>
+        <div class="ui-loading-error is-hidden">
+            <i class="fa fa-exclamation-triangle message-error" aria-hidden=true></i>
+            <span class="copy"><%- gettext("An error occurred. Please reload the page.") %></span>
+        </div>
+        <div class="wrapper-profile-section-two">
+            <div class="profile-section-two-fields">
+                <% if (!showFullProfile) { %>
+                    <span class="profile-private--message"><%- gettext('This edX learner is not currently sharing their profile details.') %></span>
```

Comment:
If you're looking at your own profile: You are currently sharing a limited profile.

If you're looking at someone else's profile: This edX learner is currently sharing a limited profile.